planning/behavior_path_planner/src/scene_module/goal_planner/goal_planner_module.cpp

```diff
@@ -1071,6 +1109,36 @@ PathWithLaneId GoalPlannerModule::generateFeasibleStopPath()
   return stop_path;
 }
 
+PathWithLaneId GoalPlannerModule::generateStopPointInCurrentPath()
+{
+  const auto & current_pose = planner_data_->self_odometry->pose.pose;
+
+  if (status_.current_lanes.empty()) {
+    return PathWithLaneId{};
+  }
+
+  // get current path
+  auto current_path = getCurrentPath();
```

Comment:
```suggestion
  auto current_path = getCurrentPath();
  if (current_path.points.empty()) {
    return PathWithLaneId{};
  }
```

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/4927/commits/22e940b8fe63cb712f6544d3130aab2a7dc799f9